morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Yes, a submenu is a menu whose parent is a menu.  The trouble is that popup is first called when it doesn't have any parent at all; see MenuItemMorph::mouseEnter.

Oh... is the contains in mouseEnter not going to work?  Because then the main menu's children will have this submenu a million times.